FHEM/14_SD_UT.pm

```diff
@@ -773,9 +774,10 @@ sub SD_UT_Attr(@) {
 			
 			my $search = "FileLog_$devicename";
 			
-			fhem("define $devicename SD_UT $devicemodel $deviceCode") if ($devicename);			# create new device
-			fhem("attr $devicename model $attrValue") if ($devicename);							# set model
-			fhem("attr $devicename room SD_UT") if ($devicename);								# set room
+			fhem("define unknown_please_select_model SD_UT unknown_please_select_model") if ($devicename eq "unknown");		# if user push attr return to unknown
+			fhem("define $devicename SD_UT $devicemodel $deviceCode") if ($devicename ne "unknown");						# create new device
+			fhem("attr $devicename model $attrValue") if ($devicename ne "unknown");										# set model
+			fhem("attr $devicename room SD_UT") if ($devicename ne "unknown");												# set room
```

Comment:
Ich habe das eben erst gesehen, davor ist mir das nicht aufgefallen.

Ich fände es besser, wenn wir hier nicht FHEM Befehle nutzen, sondern direkt in Perl bleibem

Zum Definieren eines Gerätes gibt es die Funktion CommandDefine
Und CommandAttr um ein Attribut zu setzen.

Das anlegen des Gerätes könnte man wie sonst auch komplett über autocreate steuern lassen. 
Spricht da irgendwas dagegen?

Reply:
Dagegen spricht nichts und wenn du es so haben möchtest werden wir es so umsetzen.
Ich setze mich mal damit auseinader und gebe dir die Vorlage um wieder was zu finden ;-)

Reply:
Du suchst dir wieder Befehle heraus die scheinbar nicht in der Wiki stehen ;-) *Herausforderungen*